webapp/templates/jury/submission.html.twig

```diff
@@ -757,6 +757,12 @@
                             {% else %}
                                 <pre class="output_text">{{ runsOutput[runIdx].output_system | parseRunDiff }}</pre>
                             {% endif %}
+
+                            {% if runsOutput[runIdx].team_message is not empty %}
+                                <h5>Judge message</h5>
```

Comment:
Since there is also a "judge message" concept in a run, might be less confusing to make explicit that this is the team message, not the thing called "judge message" in the output. Something like "(Judge) message to team"?